Defs/Ammo/Advanced/80x256mmFuelCell.xml

```diff
@@ -75,10 +75,10 @@
 		</graphicData>
 		<projectile Class="CombatExtended.ProjectilePropertiesCE">
```

Comment:
Wasn't reducing the "direct" damage of the 80x256mm shells the whole idea of switching it from thermobaric to incendiary in the first place?

Reply:
The direct bomb damage was previously applied in the full radius of the ammo, it's a small explosion now. This change is to alleviate the concern of the ammo type being undertuned especially on a direct hit.